geode-connectors/build.gradle

```diff
@@ -33,15 +33,56 @@ dependencies {
   integrationTestCompile(project(':geode-dunit')) {
     exclude module: 'geode-core'
   }
+  integrationTestCompile(project(':geode-junit')) {
+    exclude module: 'geode-core'
+  }
+  distributedTestCompile(project(':geode-junit')) {
+    exclude module: 'geode-core'
+  }
   acceptanceTestCompile(project(':geode-dunit')) {
     exclude module: 'geode-core'
   }
+  acceptanceTestCompile(project(':geode-junit')) {
+    exclude module: 'geode-core'
+  }
+
+  compile('com.zaxxer:HikariCP:3.2.0')
```

Comment:
Sounds good!

For what it's worth, I'm hoping to finish up a `spotlessGradle` ruleset that will enforce grouping in dependency declarations.  It has a bit of confusion around comments, though.  But hopefully a PR for it will be up soon.

Reply:
Wait, I'm confusing myself on how I should parse your comment...

Which do you find preferable?  Configuration > Project > Alphabetical, or Project > Configuration > Alphabetical?  E.g.,

Option A:

```
compile project('A')
testCompile project('B')

compile 'other-thing'
testCompile 'other-other-thing'
```

Option B:

```
compile project('A')
compile 'other-thing'

testCompile project('B')
testCompile 'other-other-thing'
```

Reply:
Option B

Reply:
I'll integrate that into my work-in-progress for a Gradle Spotless, then!